liquibase-core/src/test/java/com/example/liquibase/parser/java/ExampleJavaChangelog.java

```diff
@@ -0,0 +1,55 @@
+package com.example.liquibase.parser.java;
+
+import liquibase.change.core.RenameTableChange;
+import liquibase.parser.core.java.ChangeSet;
+import liquibase.precondition.core.PreconditionContainer;
+import liquibase.precondition.core.TableExistsPrecondition;
+
+public class ExampleJavaChangelog {
+
+    @ChangeSet(id = "1", author = "nvoxland", order = 1)
+    public void firstChange() {
+        System.out.println("Ran first change!");
+    }
+
+    @ChangeSet(id = "2", author = "nvoxland", order = 2)
+    public void secondChange() {
+        System.out.println("Ran second change!");
+    }
+
+
+    @ChangeSet(id = "3", author = "nvoxland", order = 3)
+    public RenameTableChange createTable() {
```

Comment:
Example of a changeset that returns a Change object. We should be able to return a Change[] or List<Change> as well and have the parser code handle that.

Many of the Change objects have a fluent style API, but it's not always consistant. Supporting this will better force me to ensure they are all following that convention.

We may actually not want this in the first revision, either. I'm renaming the "Change" class to "Action" and restructuring it a bit in 3.8 and that would break people writing changelogs using Change. But, wanted to have it here as an example of what we will want to support